lnbits/wallets/clightning.py

```diff
@@ -84,9 +86,17 @@ async def create_invoice(
             error_message = f"lightningd '{exc.method}' failed with '{exc.error}'."
             return InvoiceResponse(False, label, None, error_message)
 
-    async def pay_invoice(self, bolt11: str) -> PaymentResponse:
+    async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
+        invoice = lnbits_bolt11.decode(bolt11)
+        fee_limit_percent = (invoice.amount_msat // fee_limit_msat) // 100
```

Comment:
This looks inverted. Also: why use `//`? `maxfeepercent`'s default is itself a `float` (0.5).

```suggestion
       fee_limit_percent = fee_limit_msat / invoice.amount_msat * 100

```

Reply:
Much better!